integration-tests/api/__tests__/store/orders.js

```diff
@@ -501,17 +501,20 @@ describe("/store/carts", () => {
         })
       )
 
-      const responseFail = await api
+      const successRes = await api
         .post(`/store/carts/${cartId}/complete`)
         .catch((err) => {
           return err.response
         })
```

Comment:
I believe we can remove the catch here. the test will fail anyway in case of an error